src/app.ts

```diff
@@ -72,15 +81,26 @@ app.post('/', async (req: Request, res: Response) => {
       res.status(400).send('External URI not allowed on this API');
       throw new Error('External data url not allowed');
     }
-    return fetch(uri, {}).then((response) => {
-      if (!response.ok) throw new Error(response.status + response.statusText);
-      return response.text();
-    });
+    return originalHttp(uri, options);
+  };
+
+  loader.load = async (url, options) => {
+    try {
+      if (options) {
+        return await originalLoad(url, {
+          ...options,
+          ...{ baseURL }
+        });
+      }
+      return await originalLoad(url, { baseURL });
+    } catch {
+      return await originalLoad(url, options);
```

Comment:
What's the default base url? I think it should be `https://vega.github.io/vega-datasets/`.

Reply:
The default baseURL is sent from the editor front end so I didn't hard code it

Reply:
If the user did not provide any base URL, shouldn't it be empty?

Reply:
No, I think if the user does not provide one, we should use `https://vega.github.io/vega-datasets/`. That's more consistent with the editor.